Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
new file mode 100644
--- /dev/null	(date 1665411132964)
+++ b/src/Main.java	(date 1665411132964)
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        System.out.println("Let's calculate the area of a triangle");
+
+        Scanner input = new Scanner(System.in);
+
+        System.out.println("Please input the base of the triangle (in inches).");
+        double base = input.nextDouble()
+
+        while (base <= 0) {
+            System.out.println("That's invalid. Please input the base of the triangle (in inches).");
+            base = input.nextDouble();
+        }
+
+        System.out.println("Please input the height of the triangle (in inches).");
+        double height = input.nextDouble();
+        while (height <= 0) {
+            System.out.println("That's invalid. Please input the base of the triangle (in inches).");
+            base = input.nextDouble();
+        }
+
+        double area = (base * height) / 2;
+        System.out.println("The area is " + height);
+
+    }
+}
